.circleci/config.yml

```diff
@@ -81,7 +81,7 @@ commands:
           name: Update and install dependencies
           command: |
             sudo apt-get update
-            sudo apt-get install -y libssl-dev cmake
+            sudo apt-get install -y libssl-dev libdw-dev cmake
```

Comment:
When I looked yesterday, apparently cmake was there because lne lf our dependencies was using boringssl, but this is not the case anymore. So I think here we can just remove it and see if the CI screams

Reply:
Let's do this in a separate PR.